src/pages/users/[id]/index.tsx

```diff
@@ -19,56 +26,62 @@ const UserProfilePage = () => {
-            User Profile
-          </h3>
-          <p className="mt-1 max-w-2xl text-sm text-gray-500 dark:text-gray-400">
-            Personal details and application.
-          </p>
-        </div>
-        <div className="border-t border-gray-200 dark:border-gray-700">
-          <dl>
-            <div className="bg-gray-50 dark:bg-gray-900 px-4 py-5 grid grid-cols-1 gap-4 sm:grid-cols-3">
-              <dt className="text-sm font-medium text-gray-500 dark:text-gray-400">
-                Full name
-              </dt>
-              <dd className="mt-1 text-sm text-gray-900 dark:text-white sm:col-span-2">
-                {user.name}
-              </dd>
-            </div>
-            <div className="bg-white dark:bg-gray-800 px-4 py-5 grid grid-cols-1 gap-4 sm:grid-cols-3">
-              <dt className="text-sm font-medium text-gray-500 dark:text-gray-400">
-                Encryption Public Key
-              </dt>
-              <dd className="mt-1 text-sm text-gray-900 dark:text-white sm:col-span-2">
-                {user.encPk}
-              </dd>
+    <div>
+      <AppBackHeader />
+      <div className="flex flex-col gap-6">
+        <div className="flex gap-6 items-center">
+          <div className="h-32 w-32 rounded bg-slate-200"></div>
+          <div className="flex flex-col gap-1">
+            <h2 className=" text-xl font-gray-12 font-light">{user.name}</h2>
+            <div className="flex items-center gap-1">
+              <Icons.checkedCircle />
+              <span className="text-sm font-light text-white">
+                Connected XXX
+              </span>
             </div>
+            <Button onClick={() => router.push(`/users/${id}/share`)}>
+              Connect
+            </Button>
+          </div>
+        </div>
+        <div className="flex flex-col gap-2">
+          {user.outTs ? (
+            <span>{`You connected with ${user.name} on ${new Date(
+              user.outTs
+            ).toLocaleString()}`}</span>
+          ) : (
+            <span>{`You have not yet connected with ${user.name}.`}</span>
+          )}
+          {user.inTs ? (
+            <span>{`${user.name} connected with you on ${new Date(
+              user.inTs
+            ).toLocaleString()}`}</span>
+          ) : (
+            <span>{`${user.name} has not yet connected with you.`}</span>
+          )}
+        </div>
+        <ListLayout className="!gap-2" label="Links">
+          <div className="flex flex-col gap-1">
             {user.x && (
-              <div className="bg-gray-50 dark:bg-gray-900 px-4 py-5 grid grid-cols-1 gap-4 sm:grid-cols-3">
-                <dt className="text-sm font-medium text-gray-500 dark:text-gray-400">
-                  Twitter
-                </dt>
-                <dd className="mt-1 text-sm text-gray-900 dark:text-white sm:col-span-2">
-                  @{user.x}
-                </dd>
-              </div>
+              <Link href={`https://x.com/${user.x}`}>
```

Comment:
We should make the input field during registration/profile update for twitter not say "twitter.com/username" it should start with an "@" and then just have a prompt for user name so this actually works.

Reply:
I think this is a good thing to bringup during the design review - we should definitely have a list of these small frontend fixes that kali can just knock out all at once